index.js

```diff
@@ -2,56 +2,69 @@
 
 const path = require('path');
 
-const copyJsRule = (jsRule, options) => ({
-    ...jsRule,
-    test: /\.js$/, // Only assume JS files since node_modules are at least already transpiled to official JS
-    include: /[\\/]node_modules[\\/]/,
-    exclude: undefined,
-    ...options,
-    use: {
-        ...jsRule.use,
-        options: {
-            ...jsRule.use.options,
-            // Force a different cacheDirectory because there's no way to add `isNodeModule` to the cacheIdentifier
-            distDir: path.join(jsRule.use.options.distDir, 'compile-node-modules'),
-            // Add `isNodeModule` to the caller so that it's accessible in babel.config.js
-            caller: {
-                ...jsRule.use.options.caller,
-                isNodeModule: true,
+const copyJsRule = (config, options, ruleOptions) => {
+    const { dir, config: { distDir } } = options;
+
+    const jsRuleIndex = config.module.rules.findIndex((rule) => rule.use && /\bnext-babel-loader\b/.test(rule.use.loader));
+    const jsRule = config.module.rules[jsRuleIndex];
+
+    if (!jsRule) {
+        throw new Error('Could not find JS rule with next-babel-loader');
+    }
+
+    const jsRuleNodeModules = {
+        ...jsRule,
+        test: /\.js$/, // Only assume JS files since node_modules are at least already transpiled to official JS
+        include: /[\\/]node_modules[\\/]/,
+        exclude: undefined,
```

Comment:
Do we need to define `exclude` since it's `undefined`?

Reply:
It's to be sure that the default exclude of next gets cleaned up, if any.

Reply:
Hmm, ok :+1: